docs/api-spec.md

```diff
@@ -351,6 +351,55 @@ token needs to be presented when creating a replica of this primary account.
 Sublease tokens can only be issued for primary accounts. If the account in question is a replica account, 400 (Bad
 Request) is returned.
 
+## GET /keppel/v1/accounts/:name/trivy\_policies
+
+If this Keppel is configured to use its bundled [Trivy security scanner](https://aquasecurity.github.io/trivy), this
+endpoint returns a list of user-defined policies for how Keppel processes reports generated by Trivy.
+On success, returns 200 and a JSON response body like this:
+
+```json
+{
+  "policies": [
+    {
+      "match_repository": ".*",
+      "only_unfixed": true,
+      "action": {
+        "ignore": true,
+        "assessment": "risk accepted: vulnerabilities without an available fix are not actionable"
+      }
+    },
+    {
+      "match_repository": "my-python-app|my-other-image",
+      "match_vulnerability": "CVE-2022-40897",
+      "action": {
+        "severity": "Low",
+        "assessment": "adjusted severity: python-setuptools cannot be invoked through user requests"
+      }
+    }
+  ]
+}
+```
+
+The following fields may be returned:
+
+| Field | Type | Explanation |
+| ----- | ---- | ----------- |
+| `policies` | array of objects | Each policy matches a certain set of vulnerabilities on images in a certain set of repositories. Policies can be used to adjust the severity of vulnerabilities or ignore them altogether in certain situations. |
+| `policies[].match_repository` | string | The policy applies to all repositories in this account whose name matches this regex. The leading account name and slash is stripped from the repository name before matching. The notes on regexes below apply. |
+| `policies[].match_vulnerability_id` | string | The policy applies to all vulnerabilities whose ID (as shown in the `VulnerabilityID` field of Trivy's JSON report format) matches this regex. The notes on regexes below apply. |
+| `policies[].only_unfixed` | bool or omitted | If true, the policy applies only to those vulnerabilities for which no fixed version is available (that is, the `FixedVersion` field is missing in Trivy's JSON report format). |
+| `policies[].action` | object | The effect that this policy will have on matching vulnerabilities reported for images in matching repositories.  |
+| `policies[].action.assessment` | string | A human-readable description of the reasoning behind this policy. |
+| `policies[].action.ignore` | bool or omitted | If true, matching vulnerabilities will be ignored when computing the aggregated vulnerability status of the respective image. This is the same effect as if `action.severity` was set to `Clean`, but the intent is clearer. |
```

Comment:
image or manifest?

Reply:
Well.... both? The manifest is the physical representation of what appears to the user as an image. I've changed the phrasing to "the respective image manifest".